.github/workflows/ci.yml

```diff
@@ -34,10 +39,12 @@ jobs:
     with: { os: ubuntu-latest, rust: nightly }
 
   clippy:
+    if: ${{ !github.event.pull_request.draft }}
```

Comment:
Did you read through the discussion here? There seemed to be some way of bypassing the CI if one merged too quickly. Did you check whether that can still happen? https://github.com/orgs/community/discussions/25722

Reply:
From the discussion: https://github.com/orgs/community/discussions/25722#discussioncomment-3248917

This is exactly what we are doing, and it works nicely. I already tried it on this PR.

Reply:
I mean what the guy writes right below: https://github.com/orgs/community/discussions/25722#discussioncomment-3248918

Reply:
I think this is an acceptable risk, as we also require at least one review.